src/aspire/denoising/denoised_src.py

```diff
@@ -47,6 +47,6 @@ def _images(self, start=0, num=np.inf, indices=None, batch_size=512):
         for istart in range(start, end + 1, batch_size):
```

Comment:
Lets try to make this more obvious for the next person by 1) adding a comment that `start` and `end` refer to the index into the source/indices, and 2) changing `istart` to `batch_start`, and `iend` to `batch_end`, 3) adding comment above L50 that says something to the effect that we are correcting for any initial `start` offset for this `_images` set.  

Maybe you have a better way to say it, but these are the notes I added to myself in the file as I was trying to figure out what the issue/PR was about.